src/core_plugins/kibana/public/discover/index.html

```diff
@@ -1,40 +1,55 @@
 <discover-app class="app-container">
+  <!-- Local nav. -->
   <kbn-top-nav name="discover" config="topNavMenu">
-    <div class="kibana-nav-info">
-      <span ng-show="opts.savedSearch.id" class="kibana-nav-info-title">
-        <span ng-bind="::opts.savedSearch.title"></span>
-        <i aria-label="Reload Saved Search" tooltip="Reload Saved Search" ng-click="resetQuery();" class="fa fa-undo small"></i>
-      </span>
-
-      <strong class="discover-info-hits">{{(hits || 0) | number:0}}</strong>
-      <ng-pluralize count="hits" when="{'1':'hit', 'other':'hits'}"></ng-pluralize>
-    </div>
-  </kbn-top-nav>
-  <navbar name="discover-search">
-    <form role="form" class="fill inline-form" ng-submit="fetch()" name="discoverSearch">
-      <div class="typeahead" kbn-typeahead="discover">
-        <div class="input-group"
-          ng-class="discoverSearch.$invalid ? 'has-error' : ''">
-          <input
-            parse-query
-            input-focus
-            kbn-typeahead-input
-            ng-model="state.query"
-            placeholder="Search..."
-            aria-label="Search input"
-            type="text"
-            class="form-control">
-          <button type="submit"
-            ng-disabled="discoverSearch.$invalid"
-            aria-label="Search">
-            <span aria-hidden="true" class="fa fa-search"></span></button>
+    <!-- Transcluded elements. -->
+    <div data-transclude-slots>
+      <!-- Breadcrumbs. -->
+      <div data-transclude-slot="topLeftCorner" class="localBreadcrumbs">
+        <div class="localBreadcrumb">
+          <span ng-show="opts.savedSearch.id" class="localBreadcrumb__emphasis">
+            <span data-test-subj="discoverCurrentQuery" ng-bind="::opts.savedSearch.title"></span>
+            <i aria-label="Reload Saved Search" tooltip="Reload Saved Search" ng-click="resetQuery();" class="fa fa-undo small"></i>&nbsp;
```

Comment:
I think it's OK for now for a couple reasons:

1. To remove this space we'd need to either create a custom style for this icon that adds margin-left, or we would need to create a general component that does the same. We're in the midst of significantly redesigning this entire "localNav" component, so there doesn't seem to be much benefit to either of these options since we'd probably end up having to delete them.
2. Icons in this context can generally be treated as typographic elements (subject to the same line-height and font-size rules), so it's OK in my book to use the same tools to apply spacing as you would with text.